acceptance/setup/pre_suite/50_install_modules.rb

```diff
@@ -4,4 +4,8 @@
 
 step "Install the puppetdb module and dependencies" do
   on master, "puppet module install puppetlabs/puppetdb"
+  if options[:type] == 'aio'
+    # patch puppetdb module for version 4.1.0
```

Comment:
Hmm, seems a bit dodgy, what are we doing this for? Can we just patch the module properly?